dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/RegistrationEndpointServices.java

```diff
@@ -235,28 +225,19 @@ private void processPost(OAuth20Provider provider, HttpServletRequest request, H
         // POST is only allowed at the direct servlet path /registration
         String clientId = extractClientId(request.getPathInfo());
         if (!OidcOAuth20Util.isNullEmpty(clientId)) {
-            String browserErrorMsg = Tr.formatMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_INVALID_REQUEST_PATH");
-            String serverErrorMsg = TraceNLS.getFormattedMessage(RegistrationEndpointServices.class,
-                    MESSAGE_BUNDLE,
-                    "OAUTH_CLIENT_REGISTRATION_INVALID_REQUEST_PATH",
-                    null,
-                    "CWWKS1425E: The registration request was made to an incorrect URI.");
-            Tr.error(tc, serverErrorMsg);
-            throw new OidcServerException(browserErrorMsg, OIDCConstants.ERROR_INVALID_REQUEST, HttpServletResponse.SC_BAD_REQUEST);
+            BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_INVALID_REQUEST_PATH");
+            Tr.error(tc, errorMsg.getServerErrorMessage());
+            throw new OidcServerException(errorMsg.getBrowserErrorMessage(), OIDCConstants.ERROR_INVALID_REQUEST, HttpServletResponse.SC_BAD_REQUEST);
         }
 
         // Parse and obtain client
         OidcBaseClient newClient = getOidcBaseClientFromRequestBody(request);
 
         if (newClient == null) {
-            String browserErrorMsg = Tr.formatMessage(tc, locales, "OAUTH_REGISTRATION_REQUEST_MISSING_CLIENT");
-            String serverErrorMsg = TraceNLS.getFormattedMessage(RegistrationEndpointServices.class,
-                    MESSAGE_BUNDLE,
-                    "OAUTH_REGISTRATION_REQUEST_MISSING_CLIENT",
-                    null,
-                    "CWWKS1463E: The OpenID Connect registration request does not contain a client. Ensure that the request body is not empty and contains a client encoded in JSON format.");
-            Tr.error(tc, serverErrorMsg);
-            throw new OidcServerException(browserErrorMsg, OIDCConstants.ERROR_INVALID_REQUEST, HttpServletResponse.SC_BAD_REQUEST);
+            BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_REQUEST_MISSING_CLIENT");
```

Comment:
The message key in the update (`OAUTH_CLIENT_REGISTRATION_REQUEST_MISSING_CLIENT`) doesn't match the original key (`OAUTH_REGISTRATION_REQUEST_MISSING_CLIENT`).